client/src/components/WorkspaceCard/WorkspaceCard.tsx

```diff
@@ -51,6 +52,7 @@ const CardBody: any = styled.div`
       "3px solid red"
     )
   }
+  box-shadow: ${(props: any) => props.isActive && "0 0 10px 5px yellow"}
```

Comment:
one of the things I spotted in #416 was that newer TypeScript doesn't like this (as `false` isn't a valid value for `box-shadow`), I think using a ternary is more explicit and will make it happier

```suggestion
  box-shadow: ${(props: any) => props.isActive ? "0 0 10px 5px yellow" : undefined}
```

Reply:
Interesting, yeah I noticed those changes in your "Update dependencies" PR.